bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsVoTableCatalog.scala

```diff
@@ -73,11 +71,13 @@ case class GemsVoTableCatalog(backend: VoTableBackend = RemoteBackend, catalog:
   private def searchCatalog(basePosition: Coordinates, criterions: List[GemsCatalogSearchCriterion], statusLogger: StatusLogger): Future[List[GemsCatalogSearchResults]] = {
     val queryArgs = for {
       c <- criterions
-      q = CatalogQuery.catalogQueryWithoutBand(basePosition, c.criterion.radiusLimits, c.criterion.magRange.some, catalog)
+      q = CatalogQuery.catalogQuery(basePosition, c.criterion.radiusConstraint, c.criterion.magConstraint, catalog)
     } yield (q, c)
 
     val qm = queryArgs.toMap
-    VoTableClient.catalogs(queryArgs.map(_._1), backend).map(l => l.map(k => GemsCatalogSearchResults(qm.get(k.query).get, k.result.targets.rows)))
+    VoTableClient.catalogs(queryArgs.map(_._1), backend).map(l => l.collect {
+      case QueryResult(q: CatalogQueryWithMagnitudeFilters, results:CatalogQueryResult) => GemsCatalogSearchResults(qm.get(q).get, results.targets.rows)
```

Comment:
I think Rob mentioned this before but I don't recall the answer.  Why the explicit typed pattern for `results`?  It looks like `results` is always a `CatalogQueryResults` anyway.